neo/rawio/spikeglxrawio.py

```diff
@@ -82,14 +82,11 @@ class SpikeGLXRawIO(BaseRawWithBufferApiIO):
 
     Notes
     -----
-    * Contrary to other implementations this IO reads the entire folder and subfolders and:
```

Comment:
Changed this because it is not clear what other implementations this is referring to, probably the ones before but then I don't think it should be here.